random_docs/Debugging monitoring stack on Openshift.md

```diff
@@ -270,7 +270,17 @@ And the output will be opened on the webwroser
 
 ## CPU Profile interpretation
 
-[ToDo]
+From the diagrams above, we can see Node Export spends most of its CPU time in collecting metrics by netClass and netDev collector.
+Especially in the netclass collector, the performance is dragged back by excessive file operations.
+
+The netclass collector walks in each directory in the sysfs mount point /sys/net/class/, read files and parse the content to numbers.
+As for network devices, reading some properties requires locking the RTNL lock in the Linux kernel, which competes for control over the network stack with kubelet that is busy setting up network for pods. 
+The busier the network stack is, the worse the performance for both node exporter and kubelet will become.
```

Comment:
Are we concluding this only with picture/perfprofile? or, you conclude this because you also have deep knowledge of the source code? If it is the second, maybe we can make a link to the source code. If the conclusion came because you made a deeper analysis (and you did). We can say something like: a deeper analysis on the source code, make us aware that........